presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -848,6 +881,111 @@ public void testExecute()
     }
 
     @Test
+    public void testExecuteWithInsert()
+            throws Exception
+    {
+        try (Connection connection = createConnection("blackhole", "blackhole")) {
+            try (Statement statement = connection.createStatement()) {
+                assertFalse(statement.execute("INSERT INTO test_table VALUES (1)"));
+
+                assertNull(statement.getResultSet());
+                assertEquals(statement.getUpdateCount(), 1);
+                assertEquals(statement.getLargeUpdateCount(), 1L);
+            }
+        }
+    }
+
+    @Test
+    public void testExecuteUpdateWithDDLStatement()
```

Comment:
Our code style is to use camel case for words (including acronyms), even if they are normally capitalized differently. So this should be `testExecuteUpdateWithDDLStatement()` (even though it looks worse)